client/src/components/CatalogMainSection/CatalogMainSection.module.scss

```diff
@@ -0,0 +1,21 @@
+$mobile: 428 px;
+$tablet: 768px;
+$laptop: 1024px;
+$desktop: 1360px;
+
+$primaryMainColor: #0499dd;
```

Comment:
you dont need those. they are already available in our mui theme created and accessible through the sx prop

Reply:
done